src/EditorFeatures/CSharpTest/CompleteStatement/CSharpCompleteStatementCommandHandlerTests.cs

```diff
@@ -0,0 +1,2068 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Linq;
+using Microsoft.CodeAnalysis.Editor.CSharp.CompleteStatement;
+using Microsoft.CodeAnalysis.Editor.UnitTests.CompleteStatement;
+using Microsoft.CodeAnalysis.Editor.UnitTests.Workspaces;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Roslyn.Test.Utilities;
+using Xunit;
+using VSCommanding = Microsoft.VisualStudio.Commanding;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.CompleteStatement
+{
+    public class CSharpCompleteStatementCommandHandlerTests : AbstractCompleteStatementTests
+    {
+        private string CreateTestWithMethodCall(string code)
+        {
+            return
+@"class C
+    {
+        static void Main(string[] args)
+        {
+            int x = 1;
+            int y = 2;
+            int[] a = { 1,2 }
+            " + code + @"
+
+            int z = 4;
+        }
+    }
+
+    static class ClassC
+    {
+        internal static int MethodM(int a, int b)
+            => a * b;
+    }
+}";
+        }
+
+        #region ArgumentListOfMethodInvocation
+
+        [WpfFact, Trait(Traits.Feature, Traits.Features.CompleteStatement)]
+        public void ArgumentListOfMethodInvocation1()
+        {
+            var code = CreateTestWithMethodCall(@"var test = ClassC.MethodM($$x, y)");
+
+            var expected = CreateTestWithMethodCall(@"var test = ClassC.MethodM(x, y);$$");
+
+            VerifyTypingSemicolon(code, expected);
```

Comment:
could you explain what this test means?  is it literally that if you typed `;` in the middle of some method call, it would jump to the end, insert the semicolon, the place the caret afterwards?

Reply:
➡️ Yes, that's exactly what it means.